apps/web/src/style/Global.tsx

```diff
@@ -17,6 +17,7 @@ const GlobalStyle = createGlobalStyle`
 
   body {
     background-color: ${({ theme }) => theme.colors.background};
+    overflow-x: hidden;
```

Comment:
what is causing this scroll bar?

Reply:
Can't find it. And not only the home page having this issue.